common/config/environment.go

```diff
@@ -14,7 +14,7 @@ import (
 
 const (
 	namerNamespace         = "common"
-	ddInfraConfigNamespace = "ddinfra"
+	DDInfraConfigNamespace = "ddinfra"
 	ddAgentConfigNamespace = "ddagent"
```

Comment:
Should be aligned and public

Reply:
Are you suggesting to make `ddAgentConfigNamespace` also public? It will be probably needed to move `ddAgentConfigNamespace` to public but I prefer to keep private until it is needed.